src/xbot/common/controls/sensors/wpi_adapters/EncoderWPIAdapter.java

```diff
@@ -2,30 +2,28 @@
 
 import edu.wpi.first.wpilibj.Encoder;
 import xbot.common.controls.sensors.XEncoder;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
 
 public class EncoderWPIAdapter implements XEncoder {
 
     Encoder adapter;
     boolean inverted;
+    public DoubleProperty distancePerPulse;
 
-    public EncoderWPIAdapter(int aChannel, int bChannel) {
+    public EncoderWPIAdapter(String name, int aChannel, int bChannel, double defaultDistancePerPulse, XPropertyManager propMan) {
         adapter = new Encoder(aChannel, bChannel);
+        distancePerPulse = propMan.createPersistentProperty(name + "-DistancePerPulse", defaultDistancePerPulse);
```

Comment:
camel casing here seems appropriate overall though - space on the dashboard is at a slight premium, especially with how many properties we have.